docker-compose.yaml

```diff
@@ -8,7 +8,7 @@ services:
     networks:
       - storage
     volumes:
-      - ./temp/esdata:/usr/share/elasticsearch/data
+      - esdata:/usr/share/elasticsearch/data
```

Comment:
Any particular reason to do this change?

Reply:
The temp folder is part of `.gitignore` file. if these changes are needed, can you also add these folders in ignore file.

Reply:
this are not a directory but docker [volume](https://docs.docker.com/storage/volumes/) name, no need to add these in gitignore

current config have some mounting issues in some machine, causing the containers unable to start
especially in mac where folders shared between docker VM and host